sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormWord.cs

```diff
@@ -4,17 +4,19 @@
 namespace Azure.AI.FormRecognizer.Models
 {
     /// <summary>
+    /// Represents a word recognized from the input document.
     /// </summary>
     public class FormWord : FormContent
     {
         internal FormWord(TextWord_internal textWord, int pageNumber)
             : base(new BoundingBox(textWord.BoundingBox), pageNumber, textWord.Text)
         {
-            Confidence = textWord.Confidence != null ? textWord.Confidence : Constants.DefaultConfidenceValue;
+            Confidence = textWord.Confidence != null ? textWord.Confidence.Value : Constants.DefaultConfidenceValue;
         }
 
         /// <summary>
+        /// Measures the degree of certainty of the recognition result. Possible values lay between [0.0, 1.0].
         /// </summary>
-        public float? Confidence { get; }
+        public float Confidence { get; }
```

Comment:
Double checking: do we want to make it non-nullable?

Reply:
Yes - it's optional from the service as an optimization, but we'll always populate it with a default value in the SDK if it's null.